presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -107,28 +107,23 @@ public static Response getLogs(String filename, JaxrsParameter<Instant> startDat
         requireNonNull(logLevel);
 
         if (!startDate.isValid() || !endDate.isValid()) {
-            return Response.status(Status.BAD_REQUEST)
-                    .entity("Invalid date format").build();
+            return badRequest("Invalid date format");
```

Comment:
I found a better solution; I can throw a `WebApplicationException` when parsing the parameters, with the desired response as the argument. I'll do that.